polaris.shopify.com/README.md

```diff
@@ -16,3 +16,75 @@ To quickly create new components, run:
 ```
 yarn create-component
 ```
+
+## Site Content
+
+Our website combines next.js pages with [MDX](https://mdxjs.com/) markdown files. Here's a guide to understanding the content directory structures and how they translate to pages within the website.
```

Comment:
```suggestion
Our website combines [Next.js](https://nextjs.org/) pages with [MDX](https://mdxjs.com/) markdown files. Here's a guide to understanding the content directory structures and how they translate to pages within the website.
```

Light suggestion to also include a link to Next.js docs like we're doing for MDX